### ISSUE TITLE
peds crash

### ISSUE THREAD

**Issue by mohmando** [dojwun/qb-multicharacter#4]
I get a crash whenever i select the third slot which means that something is wrong with the male ped np_m_character_select , i can't also find the ydd file for the np_m_character_select

**Comment by dojwun** [comment]
Yeah ive updated it so I might have forgot something, ive been thinking about combining the map and peds together so i might just do that

**Comment by dojwun** [comment]
ive added everything, you can give it a try
